Podfile

```diff
@@ -7,6 +7,7 @@ inhibit_all_warnings!
 target :Awful do
   pod 'AFNetworking', '2.5.0'
   pod 'ARChromeActivity', '1.0.4'
+  pod 'AsyncDisplayKit', '1.1.1'
```

Comment:
Sweet! Pinning to 1.1.1 is by no means a long-term thing.

In general I specify exact versions in my Podfile because I don't trust maintainers to actually follow semver, but I'm pretty aggressive in bumping to the latest version after perusing release notes.